languages/README.md

```diff
@@ -1,10 +0,0 @@
-Languages
```

Comment:
We also need to remove?

https://github.com/WordPress/gutenberg/blob/2fc34bc151bd53b72dc9a5fb7d4112be3ae8828c/babel.config.js#L19-L24

Reply:
Yes, good point! I didn‘t notice it was there.

Reply:
And these? From a search of "languages/"

https://github.com/WordPress/gutenberg/blob/2fc34bc151bd53b72dc9a5fb7d4112be3ae8828c/.gitignore#L7-L8

https://github.com/WordPress/gutenberg/blob/2fc34bc151bd53b72dc9a5fb7d4112be3ae8828c/bin/build-plugin-zip.sh#L121-L122

Reply:
Yep, to be removed as well.